lib/models/speech-credential.js

```diff
@@ -23,10 +23,11 @@ class SpeechCredential extends Model {
   static async getSpeechCredentialsByVendorAndLabel(service_provider_sid, account_sid, vendor, label) {
     let sql;
     if (account_sid) {
-      sql = `SELECT * FROM speech_credentials WHERE account_sid = ? AND vendor = ? ${label ? 'AND label = ?' : ''}`;
+      sql = `SELECT * FROM speech_credentials WHERE account_sid = ? AND vendor = ?
+        AND label ${label ? '= ?' : 'is NULL'}`;
     } else {
       sql = `SELECT * FROM speech_credentials WHERE service_provider_sid = ? AND vendor = ?
-        ${label ? 'AND label = ?' : ''}`;
+        AND label ${label ? '= ?' : 'is NULL'}`;
```

Comment:
is this a fix for a different bug?

Reply:
Yes this is for fixing issue that, When we ask tts and stt model for vendor elevenlabs without any label

Api server will run a query without checking label, so the query might return a list of available speech credential for same vendor, api server then choose the 1st object in the list which is wrong to do next task

When no label is provided, we just want to take speech credential with vendor is NULL

Reply:
there was a similar issue reported but using whisper tts.  Would this fix that as well?

Reply:
I think yes, could you link the issue to this PR?

Reply:
https://jambonz.freshdesk.com/a/tickets/296

Reply:
No this issue is maybe something on feature server and not this play.